### ISSUE TITLE
T2 patchy biases

### ISSUE THREAD

**Issue by jesusff** [CORDEX-WRF-community/euro-cordex-cmip6#2]
This is a patch agreed to be applied to the new EURO-CORDEX WRF simulations during the meeting on Feb 9th, 2023.

The problem is described in this [WRF forum post](https://forum.mmm.ucar.edu/threads/strange-t2-bias-in-newer-wrf-versions.12259), although it is still not clear how to fix it. Initially, it was thought to be related to a coefficient ([0.02](https://github.com/NCAR/noahmp/blob/8a8073e6/src/module_sf_noahmplsm.F#L4050)), which is suggested in the forum to be lowered to 0.002. This coefficient is [now read from the MPTABLE](https://github.com/NCAR/noahmp/blob/3be0b286/src/module_sf_noahmplsm.F#L4051) ([CBIOM](https://github.com/NCAR/noahmp/blob/3be0b286/parameters/MPTABLE.TBL#L70)), which facilitates the change and even allows for a different coefficient for different land use types.

However, a series of recent bugs detected in Noah-MP also affect this line of code, so the root of the problem is more likely on these bugs, which are related to a lack of scaling of several quantities with the vegetation fraction (FVEG). This would affect differently landuse types with deciduous vegetation, which show low vegetation fractions during summer. The T2 biases in the original WRF post have a seasonal behaviour which could be related to this lack of scaling.

 * https://github.com/NCAR/noahmp/issues/86
 * https://github.com/NCAR/noahmp/issues/91
 * https://github.com/NCAR/noahmp/issues/92

**Comment by jesusff** [comment]
The new CWC WRF [v4.5.1.2](https://github.com/CORDEX-WRF-community/WRF/releases/tag/v4.5.1.2) has now fixes for all these bugs.
CBIOM changes should not be necessary, but we might reopen the issue if T2 patches remain.

**Comment by yoselita** [comment]
It looks like these colds pools not disappear with all the updates in Noah-MP that are made recently. We were checking the nature of these cold pools, and it seems that they do not appear from the beginning of the simulations. The simulation starts ok, with normal values for the surface variables and 3D variables in lower layers. But, after a very cold event happens, during a nighttime, when very thin and stable PBL is formed, these cold pools appear and remain for some time. It looks like they disappear when the next cold event happens (this is something we noticed in the our CP simulations for FPS-URBAN). Furthermore, these cold pools appear in different locations, and at different times, but they are more pronounced in the warmer months. 

It seems that it is necessary to change CBIOM values in MPTABLE.TBL from 0.02 to 0.002 after all. When done so, the patches disappear. 

I run 2 simulations using the same restart file for 01.05.2016, and the patches are wiped out after few timesteps with CBIOM set to 0.002. I changed the CBIOM value from 0.02 to 0.002 for all the LU categories, as this cannot be linked to LU map. 

![image](https://github.com/CORDEX-WRF-community/euro-cordex-cmip6/assets/54983612/86d37b81-3c7c-462e-a78a-4d650aa7cf28)
